### PR TITLE
Feature View

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
### In this PR, I did the following 👇 

- Incorporated the visual elements, layout, and styling as per the wireframes.
- Styled the wireframes and implemented the required functionalities.
- Applied CSS styles to enhance the visual appeal and usability of the application.
- Fixed linter errors

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @Jawad-Nazari 🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here is a key point to consider when documenting your README.

- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. You are not supposed to delete any of the main headers from the README template except for the optional FAQ section. 

Kindly restore and document the `Acknowledgments` section that is missing from your template.
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Jawad-Nazari** [comment]
Hi @KDlamini ,
 Thanks for your advice. I have updated the Read me file and added the the Acknowledgment section.

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Jawad-Nazari,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### The live demo and deployment links on your Table of Contents are broken. Kindly fix them or remove them if you have no sections for them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 class PostsController < ApplicationController
   def index
+    @posts = Post.where(author_id: params[:user_id])
+    @user = User.find(params[:user_id])
   end
 
   def show
+    @post = Post.find(params[:post_id])
```
Comment (TedLivist) [inline app/controllers/posts_controller.rb]: - [ ] This should be `params[:id]`, not `params[:post_id]` to avoid errors.

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Jawad-Nazari,

Good job so far!
✔️ Clickable user on index page
✔️ Implementation of design layouts
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Jawad-Nazari) [reply app/controllers/posts_controller.rb]: Hi @TedLivist ,
Thanks for revewing my code . You are right , that was my mistake and I have fixed now. 🙏

**Review by BregornOriginal — APPROVED** [review]
Hi @Jawad-Nazari 

Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! 

giphy (7)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good PR description
✅ Excellent work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**